sematic/db/models/run.py

```diff
@@ -126,3 +142,16 @@ def strip_description(self, key, value) -> str:
             value = re.sub(r"\n\s{4}", "\n", value.strip())
 
         return value
+
+    @property
+    def external_jobs(self) -> Tuple[ExternalJob, ...]:
+        """Representations of the external compute jobs used for the run."""
+        encodables = self.external_jobs_json
+        encodables = encodables if encodables is not None else []
+        return tuple(value_from_json_encodable(job, ExternalJob) for job in encodables)
+
+    @external_jobs.setter
+    def external_jobs(self, jobs: Union[List[ExternalJob], Tuple[ExternalJob, ...]]):
```

Comment:
Is the `Union` that necessary? Can just use `List` or `Sequence`?

Reply:
Sequence works